config/routing.yml

```diff
@@ -1,3 +1,9 @@
+# Dynamic routing for Pages 2.0.x
+phpbb_page_dynamic_controller:
```

Comment:
@Nicofuma Why do we require this? Can't we just do it in the core and just have it as a taggable service for extensions? Also, why is the type name `pages_extension`? That's very specific. Couldn't we make it more generalised in phpBB core?